### PR TITLE
Fix/search result error

### LINKED ISSUE #60: Typing in the search box shows "Keine Ergebnisse für Suche"
This text appears as soon as you start typing.

It should only appear when a search has been submitted and no results were found.

IMG_20200331_091510

### PR DESCRIPTION
**Pull request by jonathan-wiens**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,98 +1,114 @@
...
+          </span>{" "}
+          <br />
+          Rette den Laden bei Dir um die Ecke. Hier kannst Du helfen.
+        </Typography>
+        <form
+          className={classes.form}
+          onSubmit={(e) => {
+            e.preventDefault();
+            fetchStoreData(result);
+            submit = true;
```
Comment (jonathan-wiens) [inline pages/index.tsx:39]: This one

File: pages/index.tsx [hunk 2]
```diff
@@ -1,98 +1,114 @@
...
-	const classes = useStyles();
-	const [kiez, setKiez] = useState('');
-	const [filteredLocations, setFilteredLocations] = useState([]);
+  const classes = useStyles();
+  const [result, setResult] = useState("");
+  const [filteredLocations, setFilteredLocations] = useState([]);
 
-	const [storeData, setStoreData] = useState([]);
+  const [storeData, setStoreData] = useState([]);
+  let submit: boolean = false;
```
Comment (jonathan-wiens) [inline pages/index.tsx:17]: This one

File: pages/index.tsx [hunk 3]
```diff
@@ -1,98 +1,114 @@
...
+            Suchen{" "}
+          </Button>
+        </form>
+      </div>
+      {storeData.length > 0 ? (
+        <>
+          <CardCarousel storeData={storeData} />
+        </>
+      ) : (
+        submit &&
```
Comment (jonathan-wiens) [inline pages/index.tsx:108]: This one

**Comment by jonathan-wiens** [comment]
Sorry about the code changes due to the formatting! Maybe we should introduce prettier for it

**Review by haugthom — APPROVED** [review]

**Pull request merged**